config/locales/en.yml

```
@@ -2672,6 +2674,28 @@ en:
           permanent_home_heading: What is a permanent home?
           subtitle: For the %{year} season, this tool will only be able to serve people with specific tax situations. Please answer the following questions to find out if you can use it.
           title: First, let's see if you can use this tool to file your %{year} Maryland taxes
+      md_had_health_insurance:
+        edit:
+          authorize_follow_up: If yes, the Comptroller’s Office will share your information and the email address on file with Maryland Health Connection.
+          authorize_share_health_information: Do you authorize the Comptroller of Maryland to share information from this tax return with Maryland Health Connection for the purpose of determining pre-eligibility for no-cost or low-cost health care coverage?
+          bullet_blindness_status: Blindness status
+          bullet_fed_agi: Your federal adjusted gross income amount from Line 1
+          bullet_filing_status: Filing status reported on your return
+          bullet_insured_status: Insured/ uninsured status of each individual included in your return
+          bullet_mailing_address: Your current mailing address, email address, and phone number
+          bullet_name_ssn: Name, SSN/ITIN, and date of birth of each individual identified on your return
+          bullet_number_in_household: Total number of individuals in your household included in your return
+          bullet_relationship_to_primary: Relationship (self, spouse, or dependent) to the primary taxpayer for each individual included in your return
```

i think rather than this bullet prefix pattern it would be better to use the list feature of i18n
```suggestion
         authorize_to_share_info:
          - Blindness status
          - Your federal adjusted gross income amount from Line 1
          - Filing status reported on your return
          - Insured/ uninsured status of each individual included in your return
          - Your current mailing address, email address, and phone number
          - Name, SSN/ITIN, and date of birth of each individual identified on your return
          - Total number of individuals in your household included in your return
          - Relationship (self, spouse, or dependent) to the primary taxpayer for each individual included in your return
```
and then in the view you just loop over them